adcc/backends/test_backends_pcm.py

```diff
@@ -0,0 +1,142 @@
+import pytest
+import unittest
+import itertools
+import adcc
+import adcc.backends
+import psi4
+import os
+
+from adcc.misc import expand_test_templates
+from adcc.testdata import static_data
+from adcc.testdata.cache import psi4_data
+from numpy.testing import assert_allclose
+from adcc.exceptions import InputError
+
+from adcc.adc_pp.environment import block_ph_ph_0_pcm
+from adcc.AdcMatrix import AdcExtraTerm
+
+# remove pyscf until implemented
+backends = [b for b in adcc.backends.available()
+            if b not in ["molsturm", "veloxchem", "pyscf"]]
+basissets = ["sto-3g", "cc-pvdz"]
+methods = ["adc1", "adc2", "adc3"]
+
+
+@pytest.mark.skipif(len(backends) == 0, reason="No backend found.")
+@expand_test_templates(list(itertools.product(basissets, methods, backends)))
+class TestPCM(unittest.TestCase):
+    def template_pcm_linear_response_formaldehyde(self, basis, method, backend):
```

Comment:
I think I'd create a Psi4-specific function here, and only expand templates on `basissets` and `methods` for the time being. We won't have the other backends supported for quite some time in the future I guess.

Reply:
unfortunately true